engine/movie/oak_speech/oak_speech.asm

```diff
@@ -159,6 +170,8 @@ OakSpeechText1:
 	text_end
 OakSpeechText2:
 	text_far _OakSpeechText2A
+	; The game does not have a "sound_cry_nidorino" command programmed in.
+	; Nidorina might have been intended here during development.
 	sound_cry_nidorina
```

Comment:
Not sure if this comment is need at all, unless we have insight into why the discrepancy happened, or if it didn't happen in the JP version or something.
```suggestion
	; The cry played does not match the sprite displayed.
	sound_cry_nidorina
```

Reply:
The discrepancy was carried over from Japanese Red, Green, and Blue.

Reply:
Okay, so the simpler or no comment then.